docs/AboutUs.adoc

```diff
@@ -37,11 +37,11 @@ Responsibilities: Data
 '''
 
 === Tay Zi Hiang Willis
-image::m133225.jpg[width="150", align="left"]
-{empty}[http://github.com/m133225[github]] [<<johndoe#, portfolio>>]
+image::willis.jpg[width="150", align="left"]
```

Comment:
The filename of the profile photo is supposed to be your github username in lower case.

Reply:
ok thanks! Will adjust accordingly